lib/modelManagers/usermodel.js

```diff
@@ -44,6 +47,22 @@ class User {
     return updatedRecord;
   }
 
+  /**
+   *
+   * @param {field} field The field to be searched.
+   * @static
+   * @memberof User
+   * @returns {object} The parameters from the query
+   */
+  static async findUser(field) {
+    const UserRecord = await user.findOne({
+      where: {
+        email: field
```

Comment:
Why are we finding a user just by email?

Reply:
The function checks if the email is available during login.

Reply:
My expectation here is that `findUser` returns a user's record base on a given query. Please refactor the function to accommodate that functionality